Misc/NEWS.d/next/Core_and_Builtins/2025-12-24-11-39-59.gh-issue-143135.3d5ovx.rst

```diff
@@ -0,0 +1 @@
+Fix when :envvar:`PYTHONINSPECT` is ``0`` not setting :data:`sys.flags.inspect`.
```

Comment:
This looks quite cumbersome. You are a native speaker, cannot you came with something better?